waiter/bin/run-using-minimesos.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
```

Comment:
A one line description and usage example would be nice in the script as comments.